saas_public/doc/index.rst

```diff
@@ -23,6 +20,5 @@ Usage
 
 * Open menu ``[[ SaaS ]] >> Templates``
 * Choose or create Template that has Template's deployment(s) with public access
-* Copy link from Fast URL field
-* Open that link in browser either as logged in or logged out user
+* Open link `//saas_public/*template_id*/create-fast-build` in browser either as logged in or logged out user. Instead of *template_id*, insert template id with public access
```

Comment:
Why it's double ``//``? The link doesn't work. Make it less technical insturction, e.g. YOUDOMAIN.example/saas_public/....